x/gamm/keeper/pool_service_test.go

```diff
@@ -296,7 +298,7 @@ func (suite *KeeperTestSuite) TestJoinPoolNoSwap() {
 				keeper := suite.App.GAMMKeeper
 				// Test the "tokenInMaxs"
 				// In this case, to get the 50 * OneShare amount of share token, the foo, bar token are expected to be provided as 5000 amounts.
-				err := keeper.JoinPoolNoSwap(suite.Ctx, suite.TestAccs[1], poolId, types.OneShare.MulRaw(50), sdk.Coins{
+				_, _, err := keeper.JoinPoolNoSwap(suite.Ctx, suite.TestAccs[1], poolId, types.OneShare.MulRaw(50), sdk.Coins{
```

Comment:
Can we add check and test ShareOutAmount and TokenIn returned here is as expected?